.gitignore

```diff
@@ -5,5 +5,6 @@ leetgo.exe
 leetgo.yaml
 completions/
 dist/
-/go/
+/cpp/
+/golang/
 /contest/
```

Comment:
You can set the `LEETGO_WORKDIR` environment variable so that leetgo automatically changes to `LEETGO_WORKDIR` when it starts. For example, set `export LEETGO_WORKDIR=out`, generated directories will go to`out` instead of the current working directory, so you don't need to add them to .gitignore.

Reply:
thanks for the info, fixed